modules/__init__.py

```diff
@@ -47,16 +49,24 @@ def search(input, sender=None, postback=False):
         entities = payload['entities']
     else:
         intent, entities = process_query(input)
+
     if intent is not None:
+
         if intent in src.__personalized__ and sender is not None:
             r = requests.get('https://graph.facebook.com/v2.6/' + str(sender), params={
                 'fields': 'first_name',
                 'access_token': os.environ.get('ACCESS_TOKEN', config.ACCESS_TOKEN)
             })
+
             if entities is None:
                 entities = {}
             entities['sender'] = r.json()
+
+        if intent == 'die':
```

Comment:
I suggest not putting this here, adding this check sets a precedent that we should check all intents and then match it to their correct module. @swapagarwal what do you think about creating a function that checks a lookup table to map intent to their respective module?